plugins/services/src/js/components/HighOrderServiceConnection.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { isSDKService } from "#SRC/js/utils/ServiceUtil";
+
+import Service from "../structs/Service";
+import Pod from "../structs/Pod";
+import ServiceConnectionContainer
+  from "../containers/service-connection/ServiceConnectionContainer";
+import SDKServiceConnectionContainer
+  from "../containers/service-connection/SDKServiceConnectionContainer";
+
+const HighOrderServiceConnection = function(props) {
```

Comment:
I remember having a rather long discussion about the name back then. Let's please not introduce any new components following this naming scheme. @nLight please file an issue to refactor the "old" bits, to avoid confusions.

Reply:
https://jira.mesosphere.com/browse/DCOS-18864

Reply:
Ok, renamed the component